recursio-client/UI/HUD.gd

```diff
@@ -249,7 +249,7 @@ func add_capture_point() -> void:
 	_capture_points.append(capture_point)
 
 	# Convert number to letter
-	var capture_point_name = char(65 + _number_of_capture_points)
+	var capture_point_name = "I".repeat(_number_of_capture_points + 1)
```

Comment:
Same here as above. Additionally move this below line 255 (`_number_of_capture_points += 1`) then the `+1` is not required

Reply:
I would tend to write our own convert function if we use more than 4 CapturePoints, until then I think it's more maintainable if both set_names are written the same.
But sure, if the +1 looks too unperformant for you, feel free to change it xD